src/main/java/com/dev/cinema/model/dto/CinemaHallResponseDto.java

```diff
@@ -0,0 +1,22 @@
+package com.dev.cinema.model.dto;
+
+public class CinemaHallResponseDto {
+    private int capacity;
```

Comment:
we should return the id in all responseDtos

Reply:
done